app/uk/gov/hmrc/transitmovementsauditing/models/AuditType.scala

```diff
@@ -52,69 +81,80 @@ object AuditType {
+  case object TraderFailedUploadEvent extends AuditType("TraderFailedUploadEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+
+  case object SubmitArrivalNotificationFailedEvent
+      extends AuditType("SubmitArrivalNotificationFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object SubmitDeclarationFailedEvent extends AuditType("SubmitDeclarationFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object ValidationFailedEvent        extends AuditType("ValidationFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object CreateMovementDBFailedEvent  extends AuditType("CreateMovementDBFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object PushNotificationFailedEvent  extends AuditType("PushNotificationFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object AddMessageDBFailedEvent      extends AuditType("AddMessageDBFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+
+  case object PushNotificationUpdateFailedEvent
+      extends AuditType("PushNotificationUpdateFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object SubmitAttachMessageFailedEvent extends AuditType("SubmitAttachMessageFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object GetMovementsDBFailedEvent      extends AuditType("GetMovementsDBFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+  case object GetMovementDBFailedEvent       extends AuditType("GetMovementDBFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+
+  case object GetMovementMessagesDBFailedEvent
+      extends AuditType("GetMovementMessagesDBFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+
+  case object GetMovementMessageDBFailedEvent
+      extends AuditType("GetMovementMessageDBFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+
+  case object PushPullNotificationGetBoxFailedEvent
+      extends AuditType("PushPullNotificationGetBoxFailedEvent", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+
+  case object CustomerRequestedMissingMovement
+      extends AuditType("CustomerRequestedMissingMovement", commonTransitConventionTraders, None, Some("CTCTradersFailed"))
+
+  case object NCTSRequestedMissingMovement extends AuditType("NCTSRequestedMissingMovement", transitMovementsRouter, None, Some("CTCTradersFailed"))
 
   case object PresentationNotificationForThePreLodgedDeclaration
-      extends AuditType("PresentationNotificationForThePreLodgedDeclaration", commonTransitConventionTraders, Some(MessageType.IE170))
+      extends AuditType("PresentationNotificationForThePreLodgedDeclaration", commonTransitConventionTraders, Some(MessageType.IE170), None)
```

Comment:
Let's keep message type and status type audit types together.

Reply:
Oh. I am sorry. I think I missed that. corrected now.